lib/Doctrine/Import/Builder.php

```diff
@@ -990,6 +1013,70 @@ private function innerBuildActAs($actAs, $level = 0, $parent = null, array &$emi
         return $build;
     }
 
+    /**
+     * Adds the columns of the used actAs behaviors to the comment block.
+     *
+     * @param class-string $className
+     * @param array $instanceOptions
+     *
+     * @throws Doctrine_Import_Builder_Exception
+     */
+    private function determineActAsColumns($className, $instanceOptions)
+    {
+        // No class specified or class does not exist.
+        if (!$className || !class_exists($className)) {
+            return;
+        }
+
+        $actAsInstance = new $className($instanceOptions);
+        $options = $actAsInstance->getOptions();
```

Comment:
We should check that the 'getOptions' method is defined by the instanciated object. As we're not checking in an interface, the object might not implement such method.

Or am I missing something here?

Reply:
good catch, as you can see in the snapshot, missing or invalid classes are not handled properly by the builder in general but, as @alquerci already mentions, the validation of valid templates is another story.

pr is updated